src/main/java/org/jenkinsci/plugins/awsdevicefarm/AWSDeviceFarmRecorder.java

```diff
@@ -282,7 +286,10 @@ public AWSDeviceFarmRecorder(String projectName,
                                  Integer jobTimeoutMinutes,
                                  Boolean ifVideoRecording,
                                  Boolean ifAppPerformanceMonitoring,
-                                 Boolean ignoreRunError ) {
+                                 Boolean ignoreRunError,
```

Comment:
it breaks binary compatibility. My suggestion would be to use `@DataBoundSetter` and a setter method instead of adding more arguments to the constructor. The fields are public, so it won't make it worse

Reply:
I have `@databoundsetter` methods in there. Does that mean I can safely remove the parameters from the ctor?